ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/NetworkProtocolVersion.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE TypeFamilies #-}
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Ouroboros.Consensus.Shelley.Ledger.NetworkProtocolVersion (
```

Comment:
@mrBliss: For this, I was simply following the example of `Ouroboros.Consensus.Byron.Ledger.NetworkProtocolVersion`. If this isn't a suitable location, would you prefer I move this module somewhere else?

Reply:
The location is fine, I think.